distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -359,6 +359,19 @@ grant {
     permission java.io.FilePermission "/jenkins/-", "read";
 }
 
+// Permissions required by the wrapper:install command
+grant codeBase "file:/org.apache.karaf.wrapper.core/org.apache.karaf.shell.core/org.apache.karaf.shell.ssh/org.apache.sshd.core" {
+    permission java.io.FilePermission "<<ALL FILES>>", "execute";
+}
+grant codeBase "file:/org.apache.karaf.wrapper.core/org.apache.karaf.shell.ssh/org.apache.sshd.core" {
```

Comment:
❗️ Because we want to keep the number of total rows in the table low, I suggest that we add the `org.apache.karaf.shell.core` bundle to this larger row and include the ALL FILES/execute permission. It's broader, but should be sufficiently secure.

@stustison do you concur, or should that single bundle be left in its own row with the ALL FILES/execute perm?

Reply:
Merged the blocks as suggested